test/test.js

```diff
@@ -3196,14 +3186,12 @@ describe('array checks', function() {
             var arr2 = [10, 6, 7, 8, 9];
             expect(is.any.sorted(arr1, arr2)).to.be.true;
             expect(is.any.sorted([arr1, arr2])).to.be.true;
-            // extended behaviour not available with 'any' interface
```

Comment:
^ These were the orphan comments. See how they don't pertain to any code as they're at the bottom of their code block.